recipes/npp+/meta.yaml

```diff
@@ -0,0 +1,157 @@
+  - url: "https://developer.download.nvidia.com/compute/nppplus/redist/libnpp_plus/windows-x86_64/libnpp_plus-windows-x86_64-0.9.0.0_cuda12-archive.zip"  # [win and x86_64 and (cuda_compiler_version or "").startswith("12")]
+    sha256: "0d82d14a057c7021c71cf5c60b5fa834269b4c14b9ffaab1305b024039344c4a"  # [win and x86_64 and (cuda_compiler_version or "").startswith("12")]
+
+build:
+  number: 0
+  skip: true  # [(cuda_compiler_version or "None") == "None"]
+  script:
+    - check-glibc lib/*.so.*  # [linux]
+    - cp -rv lib/ $PREFIX  # [linux]
+    - cp -rv include/ $PREFIX  # [linux]
+    - xcopy lib %PREFIX%\Library\lib /E /I /Y /V  # [win]
+    - xcopy bin %PREFIX%\Library\bin /E /I /Y /V  # [win]
+    - xcopy include %PREFIX%\Library\include /E /I /Y /V  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - {{ stdlib('c') }}
+    - cf-nvidia-tools 1  # [linux]
+
+outputs:
+
+  - name: libnpp-plus{{ version.split(".")[0] }}
+    build:
+      ignore_run_exports_from:
+        - {{ compiler("cuda") }}
+    files:
+      - lib/libnpp_plus_*.so.*            # [linux]
+      - Library\bin\npp_plus_*.dll       # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cuda") }}
+        - {{ stdlib("c") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_compiler_version }}
+      run:
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libnpp_plus_c.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -f $PREFIX/lib/libnpp_plus_c.so.{{ version }}                                          # [linux]
+        - if not exist %LIBRARY_BIN%\npp_plus_c64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      summary: The NVIDIA NPP+ runtime library.
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      description: >-
+        This is a runtime package only. Developers should install libnpp-plus-dev to build
+        with NPP+.
+
+  - name: libnpp-plus
+    requirements:
+      host:
+        - {{ pin_subpackage("libnpp-plus" ~ version.split(".")[0], exact=True) }}
+      run:
+        - {{ pin_subpackage("libnpp-plus" ~ version.split(".")[0], exact=True) }}
+    about:
+      summary: The NVIDIA NPP+ runtime library.
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      description: >-
+        This is a runtime package only. Developers should install libnpp-plus-dev to build
+        with NPP+.
+
+  - name: libnpp-plus-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libnpp-plus", max_pin=None) }}
+        - {{ pin_subpackage("libnpp-plus" ~ version.split(".")[0], max_pin=None) }}
```

Comment:
What is ABI compatibility policy for this library? I assume it uses libtool versioning for the library version. i.e. libraries with the same soname are ABI compatible.

Reply:
As noted offline, the `0.` is indicating this is unstable

So think we need to pin tightly for now